cirq_superstaq/custom_gates.py

```diff
@@ -445,7 +446,7 @@ def phi(self) -> float:
     def theta(self) -> float:
         return self.exponent * np.pi
 
-    def __pow__(self, power: float) -> "RGate":
+    def __pow__(self, power: cirq.TParamVal) -> "RGate":
```

Comment:
i think this one should be reverted - `RGate.__init__` only accepts floats (for now)
```suggestion
    def __pow__(self, power: float) -> "RGate":
```

Reply:
should this have been resolved? it seems like either this signature or `RGate.__init__`'s is wrong

Reply:
Applied your suggestion. Mypy still passes.